apps/web/src/containers/StakeNOTE/sidebars/cooldown.tsx

```diff
@@ -0,0 +1,142 @@
+  const { isReadOnlyAddress, onSubmit } = useTransactionStatus(Network.mainnet);
+  const { setWalletSideDrawer, clearWalletSideDrawer } = useSideDrawerManager();
+  const account = useAccountDefinition(Network.mainnet);
+  const stakeNoteStatus = account?.stakeNOTEStatus;
+  const redeemWindowBeginDate = stakeNoteStatus?.redeemWindowBegin
+    ? new Date(stakeNoteStatus.redeemWindowBegin * 1000)
+    : null;
+
+  const handleConnectWallet = () => {
+    if (sideDrawerOpen) {
+      clearWalletSideDrawer();
+    }
+    if (!sideDrawerOpen) {
+      setWalletSideDrawer(SETTINGS_SIDE_DRAWERS.CONNECT_WALLET);
+    }
+  };
+
+  const handleSubmit = useCallback(async () => {
+    if (isReadOnlyAddress || !account) return;
+    const populatedTxn = await SNOTEWeightedPool.sNOTE_Contract
+      .connect(getProviderFromNetwork(Network.mainnet))
+      .populateTransaction.stopCoolDown();
+
+    onSubmit('StopSNOTECooldown', populatedTxn);
+  }, [isReadOnlyAddress, account, onSubmit]);
+
+  return (
+    <TransactionSidebar
+      riskComponent={<div />}
+      context={context}
+      showDrawer
+      mobileTopMargin={theme.spacing(16)}
+      showActionButtons={false}
+    >
+      {redeemWindowBeginDate && (
+        <Box>
+          <CountdownCards futureDate={redeemWindowBeginDate} variant="small" />
+          <Caption>
+            <FormattedMessage
+              defaultMessage={
+                'Your position is locked during cooldown with the option to cancel anytime.'
+              }
+            />
+          </Caption>
+        </Box>
+      )}
+      <Box>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Body
+            sx={{
+              color: theme.palette.typography.light,
+              marginBottom: theme.spacing(2),
+            }}
+          >
+            <FormattedMessage
+              defaultMessage={'Redemption window will start:'}
+            />
+          </Body>
+          <LabelValue>
+            {stakeNoteStatus?.redeemWindowBegin
+              ? getDateString(stakeNoteStatus.redeemWindowBegin, {
+                  showTime: true,
+                  slashesFormat: true,
+                })
+              : '-'}
+          </LabelValue>
+        </Box>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Body
+            sx={{
+              color: theme.palette.typography.light,
+              marginBottom: theme.spacing(2),
+            }}
+          >
+            <FormattedMessage defaultMessage={'Redemption window will end:'} />
+          </Body>
+          <LabelValue>
+            {stakeNoteStatus?.redeemWindowEnd
+              ? getDateString(stakeNoteStatus.redeemWindowEnd, {
+                  showTime: true,
+                  slashesFormat: true,
+                })
+              : '-'}
+          </LabelValue>
+        </Box>
+      </Box>
+      {isReadOnlyAddress || !account ? (
```

Comment:
Should we make this a shared component?

Reply:
It's not worth it imo. Only the date portion would be reusable and they're only used here and in the unstake drawer in portfolio. If the countdown was the same and the container styling was the same I think it would be worth it.

Reply:
Ok